server.rb

```diff
@@ -26,6 +26,7 @@
   end
 
   # always return json
+  comments = react_version = JSON.parse(File.read('./_comments.json'))
```

Comment:
This should probably be before the handling of POST (we could also drop the global `comments` and `react_version`)